src/routes/appRouter.tsx

```diff
@@ -0,0 +1,107 @@
+import { Link, createBrowserRouter } from 'react-router-dom';
+
+import MainPage from '@/pages/MainPage';
+import ProjectPage from '@/pages/project/ProjectPage';
+import ProjectDetailPage from '@/pages/project/ProjectDetailPage';
+import ProjectPostingPage from '@/pages/project/ProjectPostingPage';
+import ProjectUpdatePage from '@/pages/project/ProjectUpdatePage';
+import DeveloperListPage from '@/pages/developer/DeveloperListPage';
+import DeveloperDetailPage from '@/pages/developer/DeveloperDetailPage';
+import DMListPage from '@/pages/dm/DMListPage';
+import DMPage from '@/pages/dm/DMPage';
+import ProfilePage from '@/pages/profile/ProfilePage';
+import SettingsPage from '@/pages/settings/SettingsPage';
+import LoginPage from '@/pages/auth/LoginPage';
+import SignupPage from '@/pages/auth/SignupPage';
+import ExtraInformationPage from '@/pages/auth/ExtraInformationPage';
+import NotFoundPage from '@/pages/error/NotFoundPage';
+import NotificationPage from '@/pages/notification/NotificationPage';
+
+// 각 페이지별 라우팅 모듈화
+export const router = createBrowserRouter([
+  {
+    path: '/',
+    element: <MainPage />,
+    errorElement: <NotFoundPage />,
+    children: [
+      {
+        index: true, // /페이지
+        element: (
+          <ul>
+            <li>
+              <Link to="/projects/1">1번 프로젝트</Link>
+            </li>
+            <li>
+              <Link to="/projects/2">2번 프로젝트</Link>
+            </li>
+            <li>
+              <Link to="/developers/1">1번 개발자</Link>
+            </li>
+            <li>
+              <Link to="/developers/2">2번 개발자</Link>
+            </li>
+            <li>
+              <Link to="/developers/2">2번 개발자</Link>
+            </li>
+          </ul>
+        ),
+      },
+      {
+        path: '/projects',
```

Comment:
[react-router](https://reactrouter.com/en/main/routers/create-browser-router#routes) 공식문서를 보니까, path 앞에 슬래시는 없어도 될 것 같아요!
다른 모든 path 들도 마찬가지고요!
```suggestion
        path: 'projects',
```

Reply:
아 이건 제 선호도때문에 이렇게 작성했습니다!! 
다른 팀원분들 생각 들어보고 바로 수정하겠습니다!